src/main/resources/l10n/JabRef_en.properties

```diff
@@ -30,6 +30,8 @@ Abbreviation=Abbreviation
 
 About_JabRef=About_JabRef
 
+Paper_Reference_Relationship_Visualization = Paper_Reference_Relationship_Visualization
```

Comment:
You need to run `./gradlew localizationUpdate` to include the string in the remaining property files.